### ISSUE TITLE
Vertical operator %v%

### ISSUE THREAD

**Issue by bblum9** [jokergoo/ComplexHeatmap#285]
Hello,

Version 1.99.7 and when I try to run:

ht_list = ht1 %v% ht2

to use vertical operator, I get an error "could not find function "%v%"

**Comment by jokergoo** [comment]
Please use version >= 1.99.0

```r
library(devtools)
install_github("jokergoo/ComplexHeatmap")
```

**Comment by bblum9** [comment]
Yes, on version 1.99.7.

**Comment by jokergoo** [comment]
Strange. I can use `%v%` on my laptop.

```
> m = matrix(rnorm(100), 10)
> Heatmap(m) %v% Heatmap(m)
```

Can you show the complete code?

**Comment by bblum9** [comment]
Ran the code you just provided and got the same error. 

Think I figured it out, when I pull up, the documentation for add_heatmap, I see the direction argument, but when I call the function, get an unused argument error. I restarted the r session, again loaded the same version (1.99.7) and now it appears to work. So even though I had updated the package, the class definitions didn't update until the r session restarted.
